scripts/wrappers/status.py

```diff
@@ -208,26 +209,28 @@ def ha_cluster_formed(info):
     yaml_short = args.yaml
 
     if wait_ready:
-        isReady = wait_for_ready(timeout)
+        is_ready = wait_for_ready(timeout)
+        if not is_ready:
+            sys.exit(1)
```

Comment:
This exit should be at the end of the script so we keep the same output.

Reply:
@ktsakalozos are we good to return 1 if `is_cluster_ready()` also False ?

Reply:
see last commit